feat/feature.go

```diff
@@ -67,10 +67,17 @@ type Range interface {
 	Len() int
 }
 
+// Feature is a Range whose coordinates are defined relative to a feature
+// location. Start and End return the coordinates of the feature relative to
+// its location which can be nil. In the latter case callers should make no
+// assumptions whether coordinates of such features are comparable.
 type Feature interface {
 	Range
+	// Name returns the name of the feature.
```

Comment:
Sorry, I missed this. Thanks for the info.
With Vim it was faster to move the lines than copy paste.